README.md

```diff
@@ -2,221 +2,80 @@
 [![npm version][2]][3] [![build status][4]][5]
 [![downloads][8]][9] [![js-standard-style][10]][11]
 
-Create performant HTML elements.
-
-## Features
-- works with [virtually every framework][adapt]
-- speeds up perceived performance
-- improves rendering performance
-- graciously falls back if APIs are not available
-- weighs `~4kb`
+`1kb` library to wrap native DOM libraries to work with DOM diffing algorithms.
 
 ## Usage
-### Defer rendering until the browser has spare time available
 ```js
-var politeElement = component({
-  placeholder: function () {
-    return html`
-      <p>lol not loaded yet</p>
-    `
-  },
-  render: function () {
-    return html`
-      <p>HOW'S IT GOING CHAP</p>
-    `
-  }
-})
-
-console.log(politeElement())
-setTimeout(function () {
-  console.log(politeElement())
-}, 1000)
+// Implementer API
+var Nanocomponent = require('nanocomponent')
+var html = require('bel')
+
+function MyButton () {
+  if (!(this instanceof MyButton)) return new MyButton()
+  this._color = null
+  Nanocomponent.call(this)
+}
```

Comment:
Nice, dropping the return from the constructor makes the API more clear I think.